Indicators/PythonIndicator.cs

```diff
@@ -15,17 +15,17 @@
 
 using Python.Runtime;
 using QuantConnect.Data;
-using QuantConnect.Data.Market;
 using System;
 
 namespace QuantConnect.Indicators
 {
     /// <summary>
-    /// Provides a wrapper for <see cref="IndicatorBase{IBaseDataBar}"/> implementations written in python
+    /// Provides a wrapper for <see cref="IndicatorBase{IBaseData}"/> implementations written in python
     /// </summary>
-    public class PythonIndicator : IndicatorBase<IBaseDataBar>
+    public class PythonIndicator : IndicatorBase<IBaseData>
```

Comment:
This looks like it _might_ break existing user code -- any analysis done into potential breaking changes here?

Reply:
We have an opportunity window since this class was implemented a couple of days ago (PR #3419). The only user code that is out there is without PlotIndicator inheritance and I have unit tests to show that we cover both.